### ISSUE TITLE
SERVER_NAME,SERVER_PASSWORD,ADMIN_PASSWORD is not working in 0.19.0

### ISSUE THREAD

**Issue by okashi-ya** [thijsvanloef/palworld-server-docker#208]
When update to 0.19.0,some enviromnent variables is not working anymore.

This is the log:
```

Success! App '2394010' fully installed.
*****CHECKING FOR EXISTING CONFIG*****
SERVER_NAME="Tres"
SERVER_DESCRIPTION="Tres"
SERVER_PASSWORD="xovgur123"
ADMIN_PASSWORD="xovgur123"
PLAYERS=32
RCON_ENABLED=true
RCON_PORT=25575
BACKUP_ENABLED=true
*****STARTING SERVER*****
**./PalServer.sh -port=8211 -queryport=27015 EpicApp=PalServer**

```
There will be no password when enter the server, and the world name is set to "Default Palworld Server"
![image](https://github.com/thijsvanloef/palworld-server-docker/assets/49012015/96e76160-9bfa-4c7c-93a8-1183a82e9c96)

SERVER_NAME,SERVER_PASSWORD,ADMIN_PASSWORD is not in the command of ./PalServer.sh XXX

in 0.18.0 it looks like:
./PalServer.sh -port=8211 -servername="XXX" -adminpassword="XXX" -queryport=27015

**Comment by thijsvanloef** [comment]
could you please share your docker-compose.yml?

**Comment by okashi-ya** [comment]
docker-compose.yaml is not change

> could you please share your docker-compose.yml?

docker-compose.yml:
```

services:
   palworld:
      image: thijsvanloef/palworld-server-docker:latest
      restart: unless-stopped
      container_name: PalWorldServer
      ports:
        - 48211:8211/udp
        - 27015:27015/udp
      environment:
         - PUID=1000
         - PGID=1000
         - PORT=8211 # Optional but recommended
         - PLAYERS=32 # Optional but recommended
         - MULTITHREADING=false
         - RCON_ENABLED=true
         - RCON_PORT=25575
         - ADMIN_PASSWORD="adminpassword"
         - COMMUNITY=true  # Enable this if you want your server to show up in the community servers tab, USE WITH SERVER_PASSWORD!
         # Enable the environment variables below if you have COMMUNITY=true
         - SERVER_PASSWORD="myserverpassword"
         - SERVER_NAME="My PalWorld"
         - SERVER_DESCRIPTION="My Only PalWorld"
         - TZ=UTC
      volumes:
         - /home/PalWorldServer/data/:/palworld/

```

**Comment by thijsvanloef** [comment]
please also share the contents of your PalWorldSettings.ini file in ` /home/PalWorldServer/data`

**Comment by bostrt** [comment]
I believe if a Server Name, Description, etc has a forward slash in it, then `sed` is not going to work. I'm having this issue as well. Sed is interpreting a forward slash in the replacement string as the sed separate. 

Here's the line that is failing for me. I also see the sed error in logging output:
```
# if [ -n "${SERVER_NAME}" ]; then     echo "SERVER_NAME=${SERVER_NAME}";     sed -E -i "s/ServerName=\"[^\"]*\"/ServerName=$SERVER_NAME/" /palworld/Pal/Saved/Config/LinuxServer/PalWorldSettings.ini; fi

SERVER_NAME="http://palworld.lol 3 | OPEN 24/7 Dedicated"
sed: -e expression #1, char 40: unknown option to `s'
```

**Comment by okashi-ya** [comment]
> please also share the contents of your PalWorldSettings.ini file in ` /home/PalWorldServer/data/Pal/Saved/Config/LinuxServer`

Sorry,this is my fault.
In the configuration of the previous version, I did not add double quotation marks in positions SERVER_NAME,SERVER_PASSWORD and ADMIN_PASSWORD,It causes configuration errors that cannot be resolved properly.
My password does contain ",", and when I forget to use double quotes, the configuration file gets corrupted after **docker compose up -d**, and it still can't be recovered when I change back to a version that uses double quotes. But when I deleted PalworldSettings.ini and restarted the container everything worked fine.


Thank you for your help!
